src/mk_graph.rs

```diff
@@ -87,6 +91,16 @@ impl SmirJson<'_> {
                             c.set_color(Color::LightGrey);
                         }
 
+                        // Set out the type information of the locals
+                        let mut local_node = c.node_auto();
+                        let mut vector: Vec<String> = vec![];
+                        vector.push(String::from("LOCALS"));
+                        for (index, decl) in body.clone().unwrap().local_decls() {
+                            vector.push(format!("{index} = {}", decl.ty));
+                        }
+                        local_node.set_label(vector.join("\\l").to_string().as_str());
```

Comment:
Maybe add an empty string at the end to get the last line with `\l`, too? 
And we could maybe set a different colour to contrast it with normal nodes.
```suggestion
                        vector.push(String::from("LOCALS"));
                        for (index, decl) in body.clone().unwrap().local_decls() {
                            vector.push(format!("{index} = {}", decl.ty));
                        }
                        vector.push("".to_string());
                        local_node.set_label(vector.join("\\l").to_string().as_str());
                        local_node.set_style(Style::Filled);
                        local_node.set("color", "palegreen3");
```
![image](https://github.com/user-attachments/assets/cb049e69-9ed9-4813-a738-9c0eb3607e4c)